pehlaschoollibrary/proj.android-studio/app/build.gradle

```diff
@@ -62,15 +62,15 @@ android {
 
 
         general {
-            assets.srcDirs = ['../../../mainapp/Resources/bookviewer', '../../../mainapp/Resources/Localized/en-US/Games/Books', '../../../mainapp/Resources/Localized/sw-TZ/Games/Books']
+            assets.srcDirs = ['../../../mainapp/Resources/bookviewer', '../../../mainapp/Resources/Localized/Games/Books', '../../../mainapp/Resources/Localized/en-US/Games/Books']
```

Comment:
Check if the assets directory is required

Reply:
Not required hence, removed.